chainer/sequential.py

```diff
@@ -260,6 +260,12 @@ def extend(self, sequential):
             self.append(layer)
 
     def insert(self, i, layer):
+        n = len(self._layers)
+        if not (-n <= i < (n + 1)):
+            raise IndexError(
+                'Index out of range: {}'.format(i))
```

Comment:
How about additionally suggesting what should be used instead? For example,
```suggestion
                'Index out of range: {}. Please use .append() method instead.'.format(i))
```

Reply:
@niboshi Doesn't it make sense?

Reply:
When user encounteres this error, I think it's unlikely that he's trying to achieve the behavior of `append`.
For example, if a sequence `seq` has 2 elements, if you want to append a new layer, you can write either `seq.append(new_layer)` or `seq.insert(2, new_layer)` and both should work.
The error is caused when trying something like `seq.insert(3, new_layer)`. I think very few people would write this way. It's more likely that the code has flawed index handling.

Reply:
Got it. Thanks for explaining.